conf/pgbackrest-repo/aws-s3-credentials.yaml

```diff
@@ -1,3 +1,4 @@
 ---
 aws-s3-key:
 aws-s3-key-secret:
+gcs-key:
```

Comment:
:thinking: Similar to the comment on the Operator side, would we want to either rename this file, or possibly create a separate one?
If renaming, perhaps something like 'cloud-storage-credentials.yaml' would be appropriate.